packages/dashboard/src/components/widgets/primitives/input/util/isValidQuery.ts

```diff
@@ -0,0 +1,7 @@
+import { AssetQuery } from '@iot-app-kit/core';
+
+// For now we will only support one SiteWise Asset Property
+const isValidQuery = (assets: AssetQuery[]) =>
+  assets?.length === 1 && assets?.[0]?.properties?.length && assets?.[0]?.properties?.length === 1;
```

Comment:
Non-blocking: Decompose conditional logic.

Reply:
Could you help me understand what your recommendation would be here?